package.json

```diff
@@ -64,8 +64,10 @@
     "karma-junit-reporter": "0.4.1",
     "karma-ng-html2js-preprocessor": "0.2.1",
     "minimatch": "3.0.0",
+    "mkdirp": "^0.5.1",
```

Comment:
Not really necessary with Node.js 5.x but has to be listed explicitly with Node.js 4.x. I created a PR https://github.com/redhat-developer-tooling/developer-platform-install/pull/160 but we can add it with this PR, it doesn't matter.